internal/base_component.go

```diff
@@ -307,3 +308,10 @@ func (base *BaseComponent) InvalidateObject(name string) {
 		base.next.InvalidateObject(name)
 	}
 }
+
+func (base *BaseComponent) GetFileBlockOffsets(options GetFileBlockOffsetsOptions) (common.BlockOffsetList, bool, error) {
```

Comment:
Can you just add a comment to specify what the bool return type means somewhere?